[15:10:04] /opt/dgl/src/runtime/tensordispatch.cc:43: TensorDispatcher: dlopen failed: /public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/dgl/tensoradapter/pytorch/libtensoradapter_pytorch_1.10.2.so: cannot open shared object file: No such file or directory
Using backend: pytorch
2022/04/01 03:10:16: before drop huge graphs, there are 1178 graphs in DD
2022/04/01 03:10:16: after drop huge graphs, there are 1126 graphs in DD
2022/04/01 03:10:16: ready to load the pre-computed wass distances
2022/04/01 03:10:18: after loading pre-computed wass distances, there are distance with w=[0.05, 0.15, 0.25, 0.35, 0.65, 0.75, 0.85, 0.95] remained
2022/04/01 03:10:18: ready to generate node embeddings
/public1/home/scfa1517/HKS-Wasserstein-kernel/signature.py:135: ComplexWarning: Casting complex values to real discards the imaginary part
  hks[i] = np.array([np.sum(np.exp(-eigenvalues*t)*eigenvectors[i]*eigenvectors[i]) for t in sample_points])
2022/04/01 03:45:33: the node embedding have been generated
2022/04/01 03:45:34: ready to compute the wass diss with sinkhorn = True and w = 0.05
2022/04/01 04:16:39: have computed the wass diss with sinkhorn = True and w = 0.05
2022/04/01 04:16:40: ready to compute the wass diss with sinkhorn = True and w = 0.15
2022/04/01 04:47:20: have computed the wass diss with sinkhorn = True and w = 0.15
2022/04/01 04:47:21: ready to compute the wass diss with sinkhorn = True and w = 0.25
2022/04/01 05:18:20: have computed the wass diss with sinkhorn = True and w = 0.25
2022/04/01 05:18:21: ready to compute the wass diss with sinkhorn = True and w = 0.35
2022/04/01 05:48:24: have computed the wass diss with sinkhorn = True and w = 0.35
2022/04/01 05:48:25: ready to compute the wass diss with sinkhorn = True and w = 0.65
/public1/home/scfa1517/anaconda3/envs/myenv/lib/python3.9/site-packages/ot/bregman.py:517: UserWarning: Sinkhorn did not converge. You might want to increase the number of iterations `numItermax` or the regularization parameter `reg`.
  warnings.warn("Sinkhorn did not converge. You might want to "
2022/04/01 06:18:26: have computed the wass diss with sinkhorn = True and w = 0.65
2022/04/01 06:18:26: ready to compute the wass diss with sinkhorn = True and w = 0.75
2022/04/01 06:48:38: have computed the wass diss with sinkhorn = True and w = 0.75
2022/04/01 06:48:39: ready to compute the wass diss with sinkhorn = True and w = 0.85
2022/04/01 07:19:34: have computed the wass diss with sinkhorn = True and w = 0.85
2022/04/01 07:19:35: ready to compute the wass diss with sinkhorn = True and w = 0.95
2022/04/01 07:55:15: have computed the wass diss with sinkhorn = True and w = 0.95
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.05_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.15_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.25_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.35_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.65_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.75_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.85_True.npy
2022/04/01 07:55:16: have saved wasserstein_distance_matrix_sinkhorn_it2_0.95_True.npy
2022/04/01 07:55:16: Wasserstein distances computation done. Saved to file.
2022/04/01 07:55:16: Running SVMs, crossvalidation: True, gridsearch: True.
2022/04/01 08:04:04: gridsearch and crossvalidation done
